src/components/Header/HeaderLink.tsx

```diff
@@ -1,23 +1,15 @@
-import styled, { css } from 'styled-components'
+import React, { ComponentProps, PropsWithChildren } from 'react'
 
-import { useTheme } from '../../hooks/useTheme'
 import { TextLink } from '../TextLink'
 
-export const HeaderLink = styled(TextLink).attrs({ target: '_blank', suffix: null })`
-  ${() => {
-    const { color, shadow, spacingByChar } = useTheme()
-    return css`
-      padding-inline: ${spacingByChar(0.25)};
-      color: ${color.TEXT_WHITE};
-      box-shadow: unset;
-
-      &:hover {
-        color: ${color.TEXT_WHITE};
-      }
-
-      &:focus-visible {
-        ${shadow.focusIndicatorStyles}
-      }
-    `
-  }}
-`
+type Props = PropsWithChildren<ComponentProps<typeof TextLink>>
+
+export const HeaderLink: React.FC<Props> = (props) => (
+  <TextLink
+    {...props}
+    href={props.href}
+    target="_blank"
+    suffix={null}
+    className="smarthr-ui-header-link shr-pe-0.25 shr-ps-0.25 shr-text-white shr-shadow-none focus-visible:shr-focus-indicator hover:shr-text-white [&:not(focus-visible)]:shr-shadow-none"
```

Comment:
- `smarthr-ui-header-link` はどこから来たのかわからなかった
- `hoge-inline` は `hoge-x` で書けます
- `[&:not(focus-visible)]:shr-shadow-none` は必要そうでした? `shr-shadow-none` が詳細度で負けてる気がしました

```suggestion
    className="shr-px-0.25 shr-text-white focus-visible:shr-focus-indicator hover:shr-text-white [&]:shr-shadow-none"
```

Reply:
@uknmr 

>smarthr-ui-header-link
残り返信するんですが、こちらは足しました！！取り急ぎ...いらないようであれば消します :+1: つける付けないって基準あるんですか？

Reply:
基準ないんですが、外から万が一スタイリングに手を入れたい場合に用意してるものなので、今回のようなコンポーネントの DOM が入れ子になっていないものは不要ですね。
書き方は定まっていて `smarthr-ui-ComponentName-subName` となってます。

Reply:
@uknmr 
>[&:not(focus-visible)]:shr-shadow-none は必要そうでした? 

はい、こちらはshr-shadow-noneだと以下のようになってしまっていたので追記しています。
shr-shadow-none単体は不要そうになったので削除しました。
![image](https://github.com/kufu/smarthr-ui/assets/25091548/f8528698-19cc-4c2b-9b81-e2b920178f15)